vertical-pod-autoscaler/enhancements/4016-in-place-updates-support/README.md

```diff
@@ -7,117 +7,175 @@
 - [Proposal](#proposal)
 - [Context](#context)
 - [Design Details](#design-details)
-    - [Applying Updates During Pod Admission](#applying-updates-during-pod-admission)
-    - [Applying Disruption-free Updates](#Applying-disruption-free-updates)
-    - [Applying Disruptive Updates](#applying-disruptive-updates)
+    - [1. Applying Updates During Pod Admission](#pod-admission)
+    - [2. In-Place Updates (**NEW**)](#in-place)
     - [Comparison of `UpdateMode`s](#comparison-of-updatemodes)
     - [Test Plan](#test-plan)
 - [Implementation History](#implementation-history)
 <!-- /toc -->
 
 ## Summary
 
-
 VPA applies its recommendations with a mutating webhook, during pod creation. It can also evict
-pods expecting that it will apply the recommendation when the pod is recreated. This is a
-disruptive process so VPA has some mechanism to avoid too frequent disruptive updates. 
+pods expecting that it will apply the recommendation when the pod is recreated. Today, this process
+is potentially disruptive as any changes in recommendations requires a pod to be recreated.
+
+We can instead reduce the amount of disruption by leveraging the [in-place update feature] which is
+currently an [alpha feature since 1.27] and graduating to [beta in 1.33].
 
-This proposal allows VPA to apply its recommendations more frequently, with less disruption by
-using the
-[in-place update feature] which is an alpha feature [available in Kubernetes 1.27.] This proposal enables only core uses
-of in place updates in VPA with intention of gathering more feedback. Any more advanced uses of in place updates in VPA
-(like applying different recommendations during pod initialization) will be introduced as separate enhancement
-proposals.
+This proposal enables only core uses of in place updates in VPA with intention of providing the
+foundational pieces. Further advanced uses of in place updates in VPA (like applying different
+recommendations during pod initialization or providing more frequent smaller updates) will be
+introduced as separate enhancement proposals.
 
 [in-place update feature]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/1287-in-place-update-pod-resources
-[available in Kubernetes 1.27.]: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.27.md#api-change-3
+[alpha feature since 1.27]: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.27.md#api-change-3
+[beta in 1.33]: https://github.com/orgs/kubernetes/projects/178/views/1
+
+### A Note On Disruptions {#disruptions}
```

Comment:
In the rendered view, the anchor link doesn't work

Reply:
Fixed, thanks!